### ISSUE TITLE
Stock bump shader can have adverse effects on lighting

### ISSUE THREAD

**Issue by auri0n** [SleepKiller/shaderpatch#115]
I've known about this for a while, but never had good before/after shots until now. Here are two screenshots: one of the Mos Eisley barge area as it appears in the shipped tat2.lvl (which uses bumpmaps) and the other of the same area, but from the ModTools version of the map (which doesn't use bumpmaps)
![tat2_bad](https://user-images.githubusercontent.com/42473604/87999248-17773f80-caaf-11ea-8a98-ba6aa6a967d6.png)
![tat2_good](https://user-images.githubusercontent.com/42473604/87999260-1cd48a00-caaf-11ea-92c6-c2d06a345578.png)
It almost looks like there's some kind of 'brightness rescaling' being performed by the bump shader, but maybe that's totally wrong.

**Comment by SleepKiller** [comment]
I'm actually unsure what could be causing this. As far as I can tell the shader responsible does handle lighting in a more or less identical fashion. Though I did do some investigations and that led me to fixing another issue with HDR Rendering.

I do see what you're talking about but I'm unsure for now if there's a possible fix for it. I will leave this issue open for now however, just in case. There are still some things I can try looking into if I find the time.

**Comment by auri0n** [comment]
No problem ;) thanks for looking into this. What's even more bizarre is that the issue isn't present for everyone; it must have something to do with GPUs/motherboards/chipsets or something like that. If I set my lighting quality to high (with shaderpatch disabled), I get a crash to desktop on Mos Eisley, Naboo and maybe a couple other maps. I've looked though the Steam support forum and it looks like only one or two other people have this problem. Who knows what he cause really is lol, I could never blame you for not being able to pinpoint something like this.
